src/lib/bcmp/dfu/bm_dfu_core.cpp

```diff
@@ -563,6 +571,24 @@ void bm_dfu_init(bcmp_dfu_tx_func_t bcmp_dfu_tx, NvmPartition * dfu_partition, c
     if(xQueueSend(dfu_event_queue, &evt, 0) != pdTRUE) {
         printf("Message could not be added to Queue\n");
     }
+
+    BcmpPacketCfg process_dfu_message = {
+      false,
+      false,
+      dfu_copy_and_process_message,
+  };
+  BmErr err = BmOK;
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUStartMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUPayloadReqMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUPayloadMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUEndMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUAckMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUAbortMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUHeartbeatMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFURebootReqMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFURebootMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFUBootCompleteMessage));
+  bm_err_check(err, packet_add(&process_dfu_message, BcmpDFULastMessageMessage));
```

Comment:
all the messages use the same processing function, so it should be okay to use the same struct for each message type